packages/patternfly-4/react-core/src/components/Tabs/TabContent.js

```diff
@@ -1,36 +1,47 @@
-import React from 'react';
+import * as React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
 
 const propTypes = {
-  /** the index of the active tab */
+  /** the index of the active tab (not needed if used outside Tabs component) */
   activeKey: PropTypes.number,
-  /** content rendered inside the tab content area. */
+  /** content rendered inside the tab content area if used inside Tabs component */
   child: PropTypes.node,
+  /** content rendered inside the tab content area if used outside Tabs component */
+  children: PropTypes.any,
+  /** class of tab content area if used outside Tabs component */
+  className: PropTypes.string,
   /** id passed from parent to identify the content section */
   id: PropTypes.string.isRequired
 };
 
 const defaultProps = {
   activeKey: 0,
-  child: null
+  child: null,
+  children: null,
+  className: null,
 };
 
-const TabContent = ({ id, activeKey, child, ...props }) => (
-  <section
-    hidden={child.props.eventKey !== activeKey}
-    className={css('pf-c-tab-content', child.props.className)}
-    id={`pf-tab-section-${child.props.eventKey}-${id}`}
-    aria-labelledby={`pf-tab-${child.props.eventKey}-${id}`}
-    role="tabpanel"
-    tabIndex="0"
-    {...props}
-  >
-    {child.props.children}
-  </section>
-);
+class TabContent extends React.Component {
+  render() {
+    const { id, activeKey, child, children, className, eventKey, innerRef, ...props } = this.props;
+    return <section
+      ref={innerRef}
+      index={eventKey}
+      hidden={children ? null : child.props.eventKey !== activeKey}
+      className={children ? css('pf-c-tab-content', className) : css('pf-c-tab-content', child.props.className)}
```

Comment:
is there no stylesheet you can import for tab content?

Reply:
For documentation's sake, there's no stylesheet (similar to the brand component), but it's useful to keep the class around (the brand component is styled by Page this way).